### DISCUSSION TITLE
I have a dumb question about the location of ~/.tabby-client/agent/config.toml

### DISCUSSION THREAD

**Discussion by jshaala1** [TabbyML/tabby#2639] [Q&A]
I am using Docker on Windows with Docker Desktop and cannot locate the config.toml file. Should this file be in the Docker installation or within the Docker image? Can someone guide me on how to find and edit this file? Thank you in advance.

**Comment by wsxiaoys** [comment]
Hi - the client side configuration is always in your host operating system environment - thus it should locates at `C:\Users\username\.tabby-client\...`

**Comment by Geremia** [comment]
@wsxiaoys `~/.tabby-client` isn't automatically being created for me.
I'm using Docker, too.
